tests/test_sourmash.py

```diff
@@ -2428,7 +2428,7 @@ def test_gather_sbt_and_sigs():
         print(out)
         print(err)
 
-        assert '0.9 kbp     100.0%  100.0%' in out
+        assert '0.9 kbp      100.0%  100.0%       0.0' in out
```

Comment:
why is there this extra space?

Reply:
There were two spaces separating the output sets and @luizirber suggested that we change it to be in the print statement. 

This: ` print_results('{:9}   {:>6}  {:>6}      {}',`
changed to :` print_results('{:9}   {:>7} {:>7} {:>9}    {}',`

Reply:
Sorry, let me re-ask :) --

your changes add a space to every output line. Why do we want an extra space in every output line? (I don't particularly mind, but it means that you have to change all the tests, and I'm not sure why it was added.)

Maybe another way to phrase it is: why did @luizirber suggest the change? :)

Reply:
Because there were two spaces between each of the formatted outputs and the length of p_query and p_match are 7 not 6 in length? So, he suggested we change it? Right @luizirber